src/mathutil.ts

```diff
@@ -0,0 +1,22 @@
+import { vec2 } from 'gl-matrix'
+
+export const lerp = (min: number, max: number, alpha: number): number => {
+  return min + alpha * (max - min)
+}
+
+/**
+ * Translate vector start by amount in a direction indicated by orientation.
+ * orientation is interpreted as clockwise radians from north (negative y).
+ */
+export const radialTranslate2 = (
+  out: vec2,
+  start: vec2,
+  orientation: number,
```

Comment:
Should `amount` come before `orientation`? I feel weird when there are two consecutive numbers that could be easily switched. The other thing to do is make them keyword args.

Reply:
Yeah, a params hash would make sense here. Having `out` and `start` more clearly separated would help as well I think. It's not always super clear why you have to pass the same value twice when this is used.

Reply:
Ok! Will make a follow-up PR to do that.